### PR TITLE
[contributing guide] added ways to contribute

### PR DESCRIPTION
**Pull request by sarah11918**
#### What kind of changes does this PR include?

- New or updated content

Added specific ways people can contribute beyond just filing issues and PRs, to highlight what we'd like people to (know they can) do!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ This document is an active work in progress! But, we hope you'll find some usefu
 
 ## Types of Contributions
 
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on PRs, adding ideas in existing GitHub Issues and Discussions and participating in our "Pinned" issue maintenance tasks!
```
Comment (ElianCodes) [inline CONTRIBUTING.md:23]:
```suggestion
You can also make a huge contribution by getting involved by leaving review comments on [PRs](https://github.com/withastro/docs/pulls), adding ideas in existing GitHub [Issues](https://github.com/withastro/docs/issues) and [Discussions](https://github.com/withastro/docs/discussions) and participating in our "Pinned" issue maintenance tasks! 
```

maybe it's a good idea to link the real GitHub issues and discussion here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +32,10 @@ We encourage you to:
 
 - **Make a PR directly** for very obvious documentation fixes like typos or broken links.
 
+- **Look at our Existing Issues** (especially those labelled `help wanted` and `good first issue`) for contributions we are actively seeking.
```
Comment (ElianCodes) [inline CONTRIBUTING.md:35]:
```suggestion
- **Look at our Existing Issues** (especially those labelled [`help wanted`](https://github.com/withastro/docs/issues?q=is:open+is:issue+label:%22help+wanted%22) and [`good first issue`](https://github.com/withastro/docs/issues?q=is:open+is:issue+label:%22good+first+issue%22)) for contributions we are actively seeking.
```

This adds links to issues open with that label

File: CONTRIBUTING.md (hunk 1 above)
Comment (sarah11918) [reply CONTRIBUTING.md:23]: Thanks! I had these when I had more bullet points explaining each one below, but then didn't want it to get too wordy so I scrapped the extra content... and lost the links in the process!

File: CONTRIBUTING.md (hunk 2 above)
Comment (sarah11918) [reply CONTRIBUTING.md:35]: I was too lazy. Thanks, Elian!

**Review by ElianCodes — APPROVED** [review]
You have no idea how long I've waited to be able to give OG Sarah a (well earned) LGTM!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,6 +16,14 @@ This document is an active work in progress! But, we hope you'll find some usefu
 
 ## Types of Contributions
 
+There are lots of ways to contribute to the Astro Docs website! 
+
+The Astro Docs website is ... an Astro website! Maintaining it requires not only written content but also maintaining Astro code and addressing a11y, CSS, UI, and UX concerns. We also make our documentation available in several languages, so we need help translating the entire site.
+
+You can also make a huge contribution by getting involved by leaving review comments on [PRs](https://github.com/withastro/docs/pulls), adding ideas in existing GitHub [Issues](https://github.com/withastro/docs/issues) and [Discussions](https://github.com/withastro/docs/discussions) and participating in our "Pinned" issue maintenance tasks! 
+
+Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
```
Comment (Yan-Thomas) [inline CONTRIBUTING.md:25]: Serial comma!
```suggestion
Every PR, especially translation PRs, needs reviewers! Reviewing PRs and leaving comments, suggestions, or an approving "LGTM!" ("Looks Good To Me!") is a great way to get started on Team Docs, and to learn more about Astro.
```

**Review by Yan-Thomas — APPROVED** [review]
Added one small suggestion, LGTM!
